Edit pass for relocated community documentation.

dharmabumstead opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->

<!---
If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->

##### ANSIBLE VERSION
<!--- Paste verbatim output from “ansible --version” between quotes below -->
```

```


##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

dharmabumstead committed:
Edit pass for relocated community documentation.

gregdek commented:
Thumbs up from me.

Some of this content is still incorrect -- e.g. the triage bot now runs more or less continually, I believe -- but we'll pick those things out in subsequent pull requests. (Hopefully at a working session at AnsibleFest.)

samdoran committed:
Updated info on Ansibullbot

Fixed improper link syntax
Change links to point to new documents
Changed Ansibot to Ansibullbot
Clarified workflow
Change formatting on commans and tags

samdoran commented:
Making a bunch of improvements to this that will definitely need some review. I believe I made the bot workflow statements more accurate as well as corrected some outdated links. It would also be nice to have @jctanner review the bot workflow to make sure it is accurate.

samdoran commented:
Ready to be reviewed.

gundalow commented on docs/docsite/rst/community/communication.rst:
```diff
@@ -5,19 +5,19 @@ Communicating
+To subscribe to a group from a non-Google account, you can send an email to the subscription address requesting the subscription. For example: ansible-devel+subscribe@googlegroups.com
 
 IRC Channel
 -----------
```
I'll update this to list all the IRC channels. 
We can then replace every other reference to mailing lists and IRC with a link to this page. 
This would include deleting the list on github.com/ansible/community

gundalow committed:
Update communication.rst

Add Contents
Use correct headings

gundalow committed:
Update communication.rst

gundalow commented:
1) Needs updating so the headings are consistent (as per style guide) 

2) I think we should add `.. contents:: Topics` on the top of any page with more than two headings, makes it easier to find info

3) Do we want anchors on any of these headings?

4) I've added the list of IRC channels. Can all other references to IRC and mailing lists be deleted and replaced with a link to this page

gundalow commented:
Can we please leave this PR open for a bit longer. I plan to hack on this during the flight to RDU. Thanks

gundalow committed:
Topics, remove whitespace, codeofconduct

dharmabumstead commented:
Absolutely.

gundalow committed:
Formatting

mattclay commented:
Closing and re-opening to trigger CI.

gundalow committed:
Correct heading

gundalow committed:
Update maintainers.rst

gregdek commented:
What do we need to do to get this pushed and live?

gregdek commented:
@dharmabumstead I think we're waiting on one last review from you.

dharmabumstead commented:
I'll do a last edit pass today. Looks like we've got merge conflicts that will need to be resolved though.

samdoran commented:
Rebased.

dharmabumstead commented:
Sweet. I'll get the edit pass done this afterlunch.


On Wed, Aug 30, 2017 at 1:18 PM, Sam Doran <notifications@github.com> wrote:

> Rebased.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ansible/ansible/pull/28368#issuecomment-326106575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmuZbd17ZA90_UyagTCW_Z5y8e7C5Wnks5sdcN8gaJpZM4O6ynY>
> .
>

dharmabumstead committed:
Minor edits and a request for more info

dharmabumstead commented on docs/docsite/rst/community/development_process.rst:
```diff
@@ -1,177 +1,114 @@
+   -  If the maintainer says ``shipit``, the pull request is labeled **shipit**, whereupon the Core team assesses it for final merge.
+   -  If the maintainer says ``needs_info``, the pull request is labeled **needs_info** and the submitter is asked for more info.
+   -  If the maintainer says **needs_revision**, the pull request is labeled **needs_revision** and the submitter is asked to fix some things.
+   -  If the pull request is labeled **needs_revision** or **needs_info** and has received comments from the submitter...TODO: what happens here?
```
This sentence was left unfinsished. What happens in this case?

gregdek replied:
This is out of date anyway and will need to be thoroughly revised when we have time. For now, let's just remove that bullet point.

dharmabumstead commented on docs/docsite/rst/community/how_can_I_help.rst:
```diff
@@ -1,33 +1,42 @@
 -------------------------------
 
-There are many forums online where Ansible users ask and answer questions. Reach out and communicate with your fellow Ansible users. Ask good questions, and give good answers. 
+There are many forums online where Ansible users ask and answer questions. Reach out and communicate with your fellow Ansible users. Ask good questions, and give good answers.
```
Are there *bad* questions? Please consider rephrasing.

gregdek replied:
There are bad questions, actually. We could consider linking to this: http://www.catb.org/esr/faqs/smart-questions.html

But for now, let's just delete that "Ask" sentence.

dharmabumstead commented on docs/docsite/rst/community/how_can_I_help.rst:
```diff
@@ -1,33 +1,42 @@
 --------------------------------
 
-There are Ansible meetups `all over the world <https://www.meetup.com/topics/ansible/>`. Join your local meetup. Attend regularly. Ask good questions. Volunteer to give a presentation about how you use Ansible. 
+There are Ansible meetups `all over the world <https://www.meetup.com/topics/ansible/>`_. Join your local meetup. Attend regularly. Ask good questions. Volunteer to give a presentation about how you use Ansible.
```
Ibid. Are there *bad* questions? Please consider not using this phrase.

gregdek replied:
Remove "bad".

dharmabumstead commented on docs/docsite/rst/community/maintainers.rst:
```diff
@@ -41,7 +45,7 @@ Issues for modules are routed to their maintainers via an automated process. Thi
 -----------
 
-Automated routing of pull requests is handled by a tool called [Ansibot](https://github.com/ansible/ansibullbot). (You could say that he moooo-ves things around.) 
+Automated routing of pull requests is handled by a tool called [Ansibot](https://github.com/ansible/ansibullbot). (You could say that he moooo-ves things around.)
```
Cute, but adding stuff like this to the documentation is creating a localization nightmare further down the road. Please remove.

gregdek replied:
OK, cut the pun.

dharmabumstead reviewed (commented):
@gregdek and @gundalow - please see comments.

dharmabumstead committed:
Removed some hard-to-localize wording.

dharmabumstead committed:
Removed incomplete sentence

dharmabumstead committed:
Minor edits per review

dharmabumstead reviewed (commented):
Edit and review finished. Good to merge.
